src/utils/logger.js

```diff
@@ -138,7 +140,7 @@ Logger.prototype._message = function (levelVal, message) {
     const output = `miew:${level}: ${message}`;
     if (level === 'error') {
       console.error(output); // NOSONAR
-    } else if (level === 'warning') {
+    } else if (level === 'warn') {
```

Comment:
Do we have a test to avoid such mistakes in the future?

Reply:
Yes, we have